src/cmake/install_tbb.cmake

```diff
@@ -187,20 +187,37 @@ if(THREADING MATCHES "^(TBB|TBB_AUTO)$" AND
                             # resolve absolute path to avoid issues with installation
```

Comment:
in this place handle only `CMAKE_SHARED_LIBRARY_SUFFIX`. Maybe the correct way is to add pattern for `CMAKE_STATIC_LIBRARY_SUFFIX` for Win32 as well?

Reply:
BTW, could you please paste a link to archive that you use for debugging?